canvas_modules/common-canvas/src/common-properties/controls/textfield/textfield.jsx

```diff
@@ -46,7 +46,8 @@ class TextfieldControl extends React.Component {
 	}
 
 	handleChange(evt) {
-		let value = evt.target.value;
+		const trimSpaces = this.props.controller.trimSpaces;
```

Comment:
We dont read values like this directly, it should be done through getters/setters